docs/release-notes/NuGet-6.9.md

```diff
@@ -0,0 +1,121 @@
+---
+title: NuGet 6.9 Release Notes
+description: Release notes for NuGet 6.9 including new features, bug fixes, and DCRs.
+author: jgonz120
+ms.author: jongonza
+ms.date: 2/1/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.9 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.9**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.9](https://visualstudio.microsoft.com/downloads/) | [8.0.200](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.9
+
+* Add `--format`, `--verbosity` , `configfile` options to `dotnet package search` - [#12978](https://github.com/NuGet/Home/issues/12978)
+
+* Add .NET 8 constants to NuGet.Frameworks  - [#12910](https://github.com/NuGet/Home/issues/12910)
```

Comment:
This only affects customers using the NuGet SDK in their own apps, so I don't think this is very interesting to most customers as a release "highlight"